_layouts/contributing.html

```diff
@@ -23,6 +23,9 @@
                 <li style="margin-left:30px" {% if page.page_submenu_hl == "edit" %} class="active" {% endif %}>
                   <a href="/contributing/write-edit-and-preview.html"><strong>- </strong> Edit and preview</a>
                 </li>
+                <li style="margin-left:30px" {% if page.page_submenu_hl == "hints" %} class="active" {% endif %}>
```

Comment:
This is probably wrong, as it compares `hints` which is a duplicate of the bellow one.

Isn't this generated? O.o